returnn/training.py

```diff
@@ -376,6 +367,24 @@ def create_returnn_config(
         if returnn_config.post_config is not None:
             post_config.update(copy.deepcopy(returnn_config.post_config))
 
+        if keep_epochs:
```

Comment:
I always prefer not to abuse Python power and keep things more readable, so here I would write explicitly
`if keep_epochs is not None`